src/utils/actionUtils.ts

```diff
@@ -116,3 +123,44 @@ export function isValidEvent(): boolean {
 export function unlinkFile(path: fs.PathLike): Promise<void> {
     return util.promisify(fs.unlink)(path);
 }
+
+async function checkVersion(app: string): Promise<string> {
```

Comment:
nit: Should this be `getVersion`? We're not doing any checks in this function, those are done in `getCompressionMethod` and `useGnuTar`